scripts/lua/header.lp

```diff
@@ -121,5 +121,5 @@ if startsWith(scriptname, 'groups') then
     <script src="<?=pihole.fileversion('vendor/bootstrap-notify/bootstrap-notify.min.js')?>"></script>
     <script src="<?=pihole.fileversion('vendor/font-awesome/all.min.js')?>"></script>
     <script src="<?=pihole.fileversion('scripts/js/utils.js')?>"></script>
-    <script src="<?=pihole.fileversion('vendor/waitMe/waitMe.min.js')?>"></script>
-    <link rel="stylesheet" href="<?=pihole.fileversion('vendor/waitMe/waitMe.min.css')?>">
+    <script src="<?=pihole.fileversion('vendor/waitMe-js/modernized-waitme-min.js')?>"></script>
+    <link rel="stylesheet" href="<?=pihole.fileversion('vendor/waitMe-js/waitMe.min.css')?>">
```

Comment:
Any reason this CSS is here? ideally it should be placed along with the rest vendor CSS files

Reply:
Have a look at the folder structure. We don't separate by Js and css anymore, but created a folder for each dependencies containing all their necessary stuff. This made it way easier to keep things up-to-date

Reply:
The point here is that CSS is better of grouped together because it helps the browser to plan ahead.

By mixing JS and CSS files we are certainly not helping. I would keep the CSS file along with the rest.